### PR TITLE
upload: fixed files list styling on mobile

### PR DESCRIPTION
**Pull request by alejandromumo**
closes https://github.com/inveniosoftware/invenio-app-rdm/issues/1735

### CODE REVIEW

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides [hunk 1]
```diff
@@ -38,13 +25,15 @@
...
     td:not(.selected-member) {
-      margin-top: .75rem;
+      margin-top: 1rem;
 
-      &:before {
-        content: attr(data-label);
-        font-weight: bold;
-        margin-right: 1rem;
-        margin-left: 2rem;
+      .file-item-size {
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides:30]: adding this as a parent selector will break all the responsive tables, no?

Comment (alejandromumo) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides]: You are right, I moved the styling rules to a more specific selector and reverted the changes on the more generic selector.

**Comment by alejandromumo** [comment]
Context:
- On mobile, the table headers are removed (using css) and data-labels are aligned with the table data. 
- Styling rules were being applied specifically for this table but could be re-used, provided a class name that is more function oriented and less context oriented.

**Comment by alejandromumo** [comment]
> Could you put a screenshot of the final result?

Of course :)

I uploadaded screenshots in the issue since the result is a product of both PRs. 

Check it here https://github.com/inveniosoftware/invenio-app-rdm/issues/1735#issuecomment-1158627891

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides [hunk 2]
```diff
@@ -27,17 +13,17 @@
...
       margin-bottom: 1rem;
       padding: 1rem !important; //overriding !important on .ui.table:not(.unstackable) tr > td
     }
 
-    &:not(.unstackable) tr > th:not(.selected-member) {
-      display: none !important;
-    }
-
-    td:not(.selected-member) {
+    td[data-label]  {
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides:26]: nice :)

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides [hunk 3]
```diff
@@ -59,14 +45,30 @@
...
-  tbody tr td {
-    &.mobile-table-cell {
       @media all and (max-width: @largestMobileScreen) {
         width: 100% !important;
       }
     }
   }
 }
+
+.ui.table.files-table {
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides:55]:
```suggestion
.ui.table.files {
```

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides [hunk 4]
```diff
@@ -59,14 +45,30 @@
...
+  border: 0;
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
+  table-layout: fixed;
+  width: 100%;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides:60]: it should work with `.fluid` from SUI?

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides [hunk 5]
```diff
@@ -59,14 +45,30 @@
...
       }
     }
   }
 }
+
+.ui.table.files-table {
+  border: 0;
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
+  table-layout: fixed;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides:59]: why is fixed layout needed?

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides (hunk 4 above)
Comment (alejandromumo) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides:60]: it does 👍

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides (hunk 5 above)
Comment (alejandromumo) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides:59]: not needed anymore when usig `.fluid`

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides (hunk 2 above)
Comment (alejandromumo) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides:26]: thanks :)

**Review by jennur — APPROVED** [review]
LGTM! Thanks for fixing this!

Just make sure the `.fluid` doesn't break anything on mobile, because I think the `.fluid`-class breaks to `width: auto` on mobile (at least it does for `.ui.container`).

**Comment by alejandromumo** [comment]
> LGTM! Thanks for fixing this!
> 
> Just make sure the `.fluid` doesn't break anything on mobile, because I think the `.fluid`-class breaks to `width: auto` on mobile (at least it does for `.ui.container`).

Thanks !

I think it works ok as well:

Screenshot at Jun 20 11-59-12

**Pull request merged**